dev-lua/lua-openssl/lua-openssl-0.7.3-r1.ebuild

```diff
@@ -39,6 +39,11 @@ src_unpack() {
 	popd > /dev/null || die
 }
 
+src_prepare() {
+	use libressl && PATCHES=("${FILESDIR}/0001-fix-libressl-compat.patch")
```

Comment:
conditional patching is bad(tm).

will it build on openssl with thel ibressl patch applied? if yes, ditch the condition.  
I can't test right now the build.

Reply:
it builds on openssl as well as on libressl. I'll dropped the condition.